tunnel/intercept/iputils.go

```diff
@@ -38,17 +38,17 @@ func getInterceptIP(hostname string, resolver dns.Resolver) (net.IP, error) {
 	addrs, err := net.LookupIP(hostname)
 	if err == nil {
 		if len(addrs) > 0 {
-			resolver.AddHostname(hostname, addrs[0].To4())
+			_ = resolver.AddHostname(hostname, addrs[0].To4())
```

Comment:
:( i'd kinda like to see at least these erorr logged but i get it - it's no worse than what we had

Reply:
I was only attempting to satify the linter. I didn't know what the correct thing to do was so I left it as is.